cmd/ci-operator/main.go

```diff
@@ -593,6 +593,10 @@ func (o *options) initializeNamespace() error {
 		updates["ci.openshift.io/ttl.hard"] = o.cleanupDuration.String()
 	}
 
+	// This label makes sure that the namespace is active, and the value will be updated
+	// if the namespace will be reused.
+	updates["ci.openshift.io/active"] = time.Now().Format(time.RFC3339)
```

Comment:
The error messages below need to be updated since TTL is not the only thing being updated anymore